src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/CompilerServices/Unsafe.cs

```diff
@@ -119,5 +119,15 @@ public static T ReadUnaligned<T>(void* source)
         {
             throw new PlatformNotSupportedException();
         }
+
+        /// <summary>
+        /// Copies bytes from the source address to the destination address.
+        /// </summary>
+        [Intrinsic]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static void CopyBlock(void* destination, void* source, uint byteCount)
+        {
+            throw new PlatformNotSupportedException();
```

Comment:
This will need to be implemented if we ever want hydration work with minimal runtime